### ISSUE TITLE
Neos Activities

### ISSUE THREAD

**Issue by Frooxius** [Frooxius/NeosPublic#152]
This has been planned for a while (originally for the beta release in May, but didn't make the cut), I'm logging it now since it's been brought up in an conversation.

Right now the issue for newcomers is that Neos can be pretty overwhelming, especially if they're looking to do a particular thing with it, since they're unguided.

To help with this, I want to implement something called "Neos Activities", which are essentially template worlds designed for particular use case - creating art, visualizing 3D models / MRI scans, generating a photo gallery, visual programming, taking notes, holding a meeting/presentation and so on.

User would simply find the right activity, click on it, which would open a world designed for that purpose (e.g. with drawing tools and some materials pre-spawned or some UI wizard / tutorial to walk the user through).

A set of these will be spawned in each user's home to get them started as well right after starting Neos. Community will be able to build more of these as well and make them available to others (potentially even sell them for NCR).

Technically they're very simple, all they need is a world as a template, name, category, description, tags and some thumbnail (ideally short animated clip), so shouldn't be too hard to implement. Creating a new activity will be pretty much just creating a Neos world as usual.

What do you guys think? Got any ideas for particular activities that should be made for a "starter pack"?

**Comment by MR-Alex42** [comment]
Art:
- Tiltbrush World
- Grafitti World (you can spray on walls/houses)
- Museum/Gallery
- Material World

Meetings:
- Conference Room
- Town Hal
- Ideation / Creativity
- Team Event

Education
- Class Room
- Lecture Hall
- Training

Architecture
- Building Design
- City Planning

Data Visualization
- Networks

Level Design
- Dungeons
- Landscapes

Games
- ...

**Comment by sirkitree** [comment]
That's a great list @MR-Alex42 - I'd be happy to help build some of these. Maybe we setup a contest and have these user generated?

**Comment by Frooxius** [comment]
Yes, those are great ideas! I love the idea of a contest, I'd very much appreciate help with these. I'll be doing some myself and adding some extra components necessary.

**Comment by sirkitree** [comment]
Submitted a simple art gallery type world called "da5id" inspired by 
![image](https://user-images.githubusercontent.com/28543/50427416-3e7e7880-085c-11e9-8570-6370ef891517.png)
